CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+- [Tracking] Introduce random `experiment_id` generation for `filestore` tracking server backend to enable threadsafe concurrency (#7070, @BenWilson2)
+- [Tracking] Remove `dataset_name` and `on_data_{name | hash}` suffixes from `mlflow.evaluate` metric keys (#7042, @harupy)
+- [Models] Remove deprecated `mlflow.keras` module and enable `keras` support within `tensorflow` flavor (#6530, @WeichenXu123)
+- [Models] Remove deprecated `mlflow.sklearn.eval_and_log_metrics` API in favor of `mlflow.evaluate` API (#6520, @dbczumar)
+- [Models] Change the default environment configuration for `spark_udf` loaded models to use `local` (#6640, @WeichenXu123)
+- [Models] Change default to `virtualenv` instead of `conda` for Models environment creation (#6459, @harupy)
+- [Models] Remove support for `pyfunc` submission to `mlflow.evaluate` (#6670, @harupy)
+- [Models] Modify `PyfuncModel` spec to support `conda` and `virtualenv` subfields (#6684, @harupy)
+- [Scoring] Introduce strict format indicators for input datatypes for the scoring API and modify the output structure of returned values (#6575, @tomasatdatabricks)
+- [Scoring] Remove public APIs for `mlflow.sagemaker.deploy()` and `mlflow.sagemaker.delete()` (#6650, @dbczumar)
+- [Scoring] Rename input argument `df` to `inputs` in `mlflow.deployments.predict()` method (#6681, @BenWilson2)
+- [Scoring] Replace the `--no-conda` cli option argument for native serving with `--env-manager='local'` (#6501, @harupy)
+- [Registry / Tracking] Remove `/preview` routes for Tracking and Model Registry REST APIs (#6667, @harupy)
+- [Projects] Remove the `use_conda` argument from `run` cli command for MLflow Projects (#6654, @harupy)
+- [Projects] Introduce support for `virtualenv` within ML Projects and set as default environment (#6489, @harupy)
+- [Projects] Introduce support for docker image creation with the `--build-image` flag for MLflow Projects (#7011, @harupy)
+- [Integrations/Azure] Remove deprecated `mlflow.azureml` modules from MLflow (#6691, @BenWilson2)
+- [R] Remove conda integration with the R client (#6638, @harupy)
+
+Features:
+
+- [Recipes] Add support for classification models to Mlflow Recipes (#7082, @bbarnes52)
+- [Tracking] Add support to `get_model_dependencies` that enables downloading of all model artifacts (#6733, @harupy)
+- [Tracking] Add support for defining `save_format` for Keras `autolog()` (#7123, @balvisio)
+- [Models] Add metric return structure and support for `custom_artifacts` to `mlflow.evaluate` (#7142, @harupy)
+- [Models] Modify behavior of defining and inferring a positive label for binary classification within `mlflow.evaluate` (#7149, @dbczumar)
+- [Models] Enable automated signature logging for `tensorflow` and `keras` models with `autolog` enabled (#6678, @BenWilson2)
+- [Models] Add support for defining `model_format` for `mlflow.xgboost` (#7068, @AvikantSrivastava)
+- [Scoring] Enable support for MLServer version 1.2.0.dev13 for MLflow 2.x (#7254, @adriangonz)
```

Comment:
This is part of the scoring API breaking change - can we add the PR to that line item instead? https://github.com/mlflow/mlflow/pull/7324/files#r1021177724